packages/circuits/src/Reputation.ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert'
+import * as crypto from '@unirep/crypto'
+
+export class Reputation {
```

Comment:
I think this class can be imported from `@unirep/core`

Reply:
Yes, it's not currently exported so I duplicated it